### PR TITLE
Mobile view first changes

### PR DESCRIPTION
**Pull request by KanoCode**
Creating the mobile version of my portfolio website to give access to mobile users.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+  </head>
+  <body>
+    <header>
+      <nav id="menu-mobile">
+        <a href="#" id="logo">John Doe</a>
+        <button type="button" class="menu-btn">
+          <i class="fa fa-bars" aria-hidden="true"></i>
+        </button>
+      </nav>
+    </header>
```
Comment (rustam-tolipov) [inline index.html]: - [x] **GOOD JOB SO FAR 🎊**, Here are some things that need changes. In this case just hamburger button should be inside `nav` tag not `a`. Please kindly fix this issue and lets move on 👌

Example
```html
<header>
  <a>....</a>
  <nav>
    your button ...
  </nav>
</header>
```

File: index.html [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <title>portfolio</title>
+  </head>
+  <body>
+    <header>
+      <nav id="menu-mobile">
+        <a href="#" id="logo">John Doe</a>
```
Comment (rustam-tolipov) [inline index.html:23]: - [x] Your second mistake is about alignment 🥲🤯. I would recommend space on the left side of your logo. **Tip 💁**: You could remove `margin-left` from `#logo` in CSS file 👍👋

File: index.html [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+        <li>
+          <a href="#"><i class="fa fa-angellist" aria-hidden="true"></i></a>
+        </li>
+        <li>
+          <a href="#"><i class="fa fa-twitter" aria-hidden="true"></i></a>
+        </li>
+        <li>
+          <a href="#"><img src="./assets/Normal-Button/Tertiary/Vector.svg" alt="icon that links to medium"></a>
+        </li>
+      </ul>
```
Comment (rustam-tolipov) [inline index.html:82]: - [ ] [OPTIONAL] I also suggest changing colors a few lines and social icons. You may see the difference below 👍⏬.

**This is your design 😐**
image

**Figma** design 😄
image

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hi @KanoCode , 

Good job so far!

### HIGHLIGHTS!

- No linters errors.💯
- Descriptive Readme.md. 💯


There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by julie-ify — APPROVED** [review]
## Hi @KanoCode, 

# Highlights
- Good job implementing the required changes from the previous reviewer ✔️ 

# Status: Approved ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# Congratulations! 🎉 

Approved emoji

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**